test/regress/cli/regress0/parser/issue9140-check-to-fp-args.smt2

```diff
@@ -0,0 +1,6 @@
+; DISABLE-TESTER: dump
+; SCRUBBER: grep -o "expected a value > 1"
+; EXPECT: expected a value > 1
+; EXIT: 1
+(set-logic ALL)
+(assert (forall ((a Real)) (= ((_ to_fp 0 1) 0.0))))
```

Comment:
Can you amend this to only have the 0 and 1 indices as the only thing that is wrong here? The equality is missing an argument and the quantifier is irrelevant.